test/calculator.test.js

```diff
@@ -0,0 +1,90 @@
+const path = require("path");
+const Web3 = require("web3");
+const { Enigma } = require("../enigmaLoader");
+const { EnigmaContractAddress, EnigmaTokenContractAddress, proxyAddress, ethNodeAddr } = require("../contractLoader");
+const constants = require("../testConstants");
+
+const { testDeployHelper, testComputeHelper } = require("../scUtils");
+
+describe("calculator", () => {
+  let accounts;
+  let web3;
+  let enigma;
+
+  beforeAll(async () => {
```

Comment:
Since we are on such a massive refactoring of this code, we may as well provide a utility function to encapsulate the contents of what is inside `beforeAll`, so that if we need to change anything in this code in the future we only have to change one file instead of many. Say for example, we want to set a different TaskKeyPair, or the version of Enigma changes.